frontend/src/hooks/pims-api/useApiProperties.ts

```diff
@@ -0,0 +1,77 @@
+import { IPropertySearchParams } from 'constants/API';
+import { IBuilding, IPagedItems, IParcel } from 'interfaces';
+import React from 'react';
+import * as pimsToasts from 'constants/toasts';
+import queryString from 'query-string';
+
+import { useApi } from '.';
+import { LifecycleToasts } from 'customAxios';
+
+/**
+ * PIMS API wrapper to centralize all AJAX requests to the property endpoints.
+ * @returns Object containing functions to make requests to the PIMS API.
+ */
+
+const parcelCreatingToasts: LifecycleToasts = {
+  loadingToast: pimsToasts.parcel.PARCEL_CREATING,
+  successToast: pimsToasts.parcel.PARCEL_CREATED,
+  errorToast: pimsToasts.parcel.PARCEL_CREATING_ERROR,
+};
+
+const parcelDeletingToasts: LifecycleToasts = {
+  loadingToast: pimsToasts.parcel.PARCEL_DELETING,
+  successToast: pimsToasts.parcel.PARCEL_DELETED,
+  errorToast: pimsToasts.parcel.PARCEL_DELETING_ERROR,
+};
+
+const parcelUpdatingToasts: LifecycleToasts = {
+  loadingToast: pimsToasts.parcel.PARCEL_UPDATING,
+  successToast: pimsToasts.parcel.PARCEL_UPDATED,
+  errorToast: pimsToasts.parcel.PARCEL_UPDATING_ERROR,
+};
+
+const buildingDeletingToasts: LifecycleToasts = {
```

Comment:
I wonder if there is a friendlier way to support different toasts without the repetitive structure here?